iothub/device/src/ClientOptions.cs

```diff
@@ -29,5 +30,27 @@ public class ClientOptions
         /// The default behavior is that <see cref="Message.MessageId"/> is set only by the user.
         /// </summary>
         public SdkAssignsMessageId SdkAssignsMessageId { get; set; } = SdkAssignsMessageId.Never;
+
+        /// <summary>
+        /// The suggested time to live value for tokens generated for SAS authenticated clients.
+        /// If unset the generated SAS tokens will be valid for 1 hour.
+        /// </summary>
+        /// <remarks>
+        /// This is used only for SAS token authenticated clients through either the
+        /// <see cref="DeviceClient.CreateFromConnectionString(string, ClientOptions)"/>/ <see cref="ModuleClient.CreateFromConnectionString(string, ClientOptions)"/> flow
```

Comment:
looks to be a stray / in there. Is that intended?

Reply:
Yes, I intend it to be displayed in the below format "option1/ option2", since I was grouping DeviceClient.CreateFromConnectionString and ModuleClient.CreateFromConnectionString together.
Do you think there is a better or clearer way to express this?

Reply:
If you wish to separate things with a slash, it should be consistent for spacing before and after.

However, you could just use the word "or" instead which I'd recommend.